stac_model/input.py

```diff
@@ -0,0 +1,52 @@
+from typing import Dict, List, Literal, Optional, Union
+
+from pydantic import AnyUrl, BaseModel, Field
+
+
+class InputArray(BaseModel):
+    shape: List[Union[int, float]]
+    dim_order: Literal["bhw", "bchw", "bthw", "btchw"]
+    data_type: str = Field(
+        ...,
+        pattern="^(uint8|uint16|uint32|uint64|int8|int16|int32|int64|float16|float32|float64)$",
+    )
+
+
+class Statistics(BaseModel):
+    minimum: Optional[List[Union[float, int]]] = None
+    maximum: Optional[List[Union[float, int]]] = None
+    mean: Optional[List[float]] = None
+    stddev: Optional[List[float]] = None
+    count: Optional[List[int]] = None
+    valid_percent: Optional[List[float]] = None
```

Comment:
It's not quite exact because we take arrays for Statistics and we only recommend including the statistics needed for normalization or other preprocessing steps. It look like this class accepts floats for the args to apply or create. so we couldn't use pystac's validation for this extension I'm assuming. is it fine then to keep using pydantic here?